lib/preprocess.js

```diff
@@ -49,7 +50,16 @@ export const preprocess = async ({
 
   const start = new Date()
   /** @type import('./typings').Measurement[] */
-  const measurements = await fetchMeasurements(cid)
+  const measurements = await pRetry(
+    () => fetchMeasurements(cid),
+    {
+      retries: 10,
+      onFailedAttempt: err => {
+        console.error(err)
+        console.error(`Retrying ${err.retriesLeft} more times`)
```

Comment:
I think it's important to include the root CID in the error message; otherwise, we wouldn't know what exactly we were retrying.